### PR TITLE
Matt electron

### PR DESCRIPTION
**Pull request by Jaitnium**
Electron has been implemented, footer files renamed, new package.json commands. 

**If the electron screen is blank, press CMD+R after the server is up (1-3 seconds)**. 

Login isn't fully implemented, but will be bypassed as long as the user is authenticated ($HOME/.aws & $HOME/.kube files).

The electron app in dev mode **will be blank**. This is because electron starts up _before_ webpack is done bundling. The electron app can be ignored entirely and devs can use the browser if that's more convenient.

### CODE REVIEW

File: client/src/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+function Login() {
+  let navigate = useNavigate();
+  const [accessKey, setAccessKey] = useState("");
+  const [secretKey, setSecretKey] = useState("");
+  const [clusterName, setClusterName] = useState("");
+  const [regionName, setRegionName] = useState("");
+
+  // If the server sends anything besides 200, then the user isn't authenticated
+  // For the user's convenience, find and parse their three files (in $HOME/.aws/ 
+  // and $HOME/.kube/) and populate their corresponding fields in the form
```
Comment (drewdunne) [inline client/src/Login.tsx:14]: Will this only work on Mac?

**Review by lejon1220 — APPROVED** [review]
Looks good

File: server/models/AuthModel.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    catch(err) {
+      // If there is an error, then the user doesn't have access to the EKS
+      return next(err);
+    }
+    // return next();
+    //const output = execSync('ls', {encoding: 'utf-8'});
+
+    // const output = execSync('kubectl get node', {encoding: 'utf-8'});
+    // console.log('Output was:\n', output);
+  }
```
Comment (drewdunne) [inline server/models/AuthModel.ts:34]: This file has a ton of comments that should probably be removed.

File: electron/main.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+const { app, BrowserWindow } = require("electron");
```
Comment (drewdunne) [inline electron/main.js:1]: Would take a pass through comments and/or console logs.

File: client/styles/theme.scss [hunk 4]
```diff
@@ -0,0 +1,133 @@
+
```
Comment (drewdunne) [inline client/styles/theme.scss:1]: Would take a pass through comments

File: client/styles/login.scss [hunk 5]
```diff
@@ -0,0 +1,59 @@
+@use "./theme" as theme;
```
Comment (drewdunne) [inline client/styles/login.scss:1]: Would take a pass through comments

File: client/src/styles/login.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
+@use "./theme" as theme;
```
Comment (drewdunne) [inline client/src/styles/login.scss:1]: Would take a pass through comments

File: client/src/Login.tsx [hunk 7]
```diff
@@ -0,0 +1,101 @@
+import React, { FormEvent, useState } from 'react';
```
Comment (drewdunne) [inline client/src/Login.tsx:1]: Would take a pass through comments and/or console logs.

**Review by drewdunne — CHANGES_REQUESTED** [review]

**Review by drewdunne — APPROVED** [review]

**Pull request merged**